pkg/client/clientcmd/client_config.go

```diff
@@ -319,8 +319,10 @@ func (inClusterClientConfig) Namespace() (string, error) {
 
 // Possible returns true if loading an inside-kubernetes-cluster is possible.
 func (inClusterClientConfig) Possible() bool {
-	fi, err := os.Stat("/var/run/secrets/kubernetes.io/serviceaccount/token")
+	tfi, terr := os.Stat("/var/run/secrets/kubernetes.io/serviceaccount/" + api.ServiceAccountTokenKey)
+	cafi, caerr := os.Stat("/var/run/secrets/kubernetes.io/serviceaccount/" + api.ServiceAccountRootCAKey)
```

Comment:
should we prevent usage if the file isn't found? why not allow an attempt to see if the cert is accepted using the default CA roots?